src/jit/gentree.h

```diff
@@ -120,6 +120,8 @@ enum genTreeKinds
     GTK_NOVALUE = 0x0400, // node does not produce a value
     GTK_NOTLIR  = 0x0800, // node is not allowed in LIR
 
+    GTK_NOCONTAIN = 0x1000, // this node is a value, but may not be contained
```

Comment:
To continue from the other PR:
Wouldn't it make more sense to have `GTK_CONTAIN` and mark only those opers that can actually be contained? 

I think that the complete oper list can be gathered from `genConsumeRegs` - if it doesn't know how to handle an oper then that oper cannot be contained. From a cursory look we have `GT_IND`, `GT_LEA`, `GT_LCL_VAR`, `GT_INIT_VAL`, all leafs and GT_SIMD (only because of SIMD 0). And if it's true that `genConsumeRegs` decides what can be contained then `canBeContained` looks like something that belongs somewhere in codegen, not in `GenTree`.